cmd/vela-terraform/terraform.go

```diff
@@ -35,6 +33,8 @@ func install(customVer, defaultVer string) error {
 		return nil
 	}
 
+	logrus.Infof("custom terraform version requested: %s", customVer)
```

Comment:
Moved this line to avoid logging unless the customVer and defaultVer are different.